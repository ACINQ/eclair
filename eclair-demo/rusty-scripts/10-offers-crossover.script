# Offers which cross over still get resolved.
A:offer 1
A:commit
B:offer 2
B:recvoffer
B:recvcommit
B:commit

A:recvoffer
A:recvrevoke
A:recvcommit
B:recvrevoke

A:commit
B:recvcommit
A:recvrevoke

checksync
echo ***A***
A:dump
echo ***B***
B:dump
