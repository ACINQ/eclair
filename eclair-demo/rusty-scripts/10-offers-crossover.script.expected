***A***
LOCAL COMMITS:
 Commit 1:
  Offered htlcs: 1
  Received htlcs: 2
  SIGNED
REMOTE COMMITS:
 Commit 2:
  Offered htlcs: 2
  Received htlcs: 1
  SIGNED
***B***
LOCAL COMMITS:
 Commit 2:
  Offered htlcs: 2
  Received htlcs: 1
  SIGNED
REMOTE COMMITS:
 Commit 1:
  Offered htlcs: 1
  Received htlcs: 2
  SIGNED
